src/lib/DefaultRuleManager.ts

```diff
@@ -78,7 +78,10 @@ export class DefaultRuleManager implements RuleManager {
 			const engineRules = rules.filter(r => r.engine === e.getName());
 			const engineTargets = await this.unpackTargets(e, targets, matchedTargets);
 			this.logger.trace(`For ${e.getName()}, found ${engineGroups.length} groups, ${engineRules.length} rules, ${engineTargets.length} targets`);
-			if (engineRules.length > 0 && engineTargets.length > 0) {
+			
+			// Not checking for selected rule count anymore since 
+			// Custom config engines do not have an upfront selection
+			if (engineTargets.length > 0) {
```

Comment:
What about `(engineTargets.length > 0 && (engineRules.length > 0 || e.isCustomConfigBased()))`?

Reply:
Good idea. Will try it out.